third-party/pom.xml

```diff
@@ -143,6 +143,12 @@
                   <transitive>false</transitive>
                   <source>false</source>
                 </artifact>
+                <!-- https://mvnrepository.com/artifact/com.google.code.findbugs/jsr305 -->
+                <artifact>
+                    <id>com.google.code.findbugs:jsr305:jar:3.0.2</id>
+                    <transitive>false</transitive>
+                    <source>false</source>
```

Comment:
Are there sources available? It would be nice to have them in the IDE for reference (even on annotations).